docker-compose-template.yml

```diff
@@ -40,7 +40,7 @@ services:
                 "clientUrl": "http://localhost:3000",
                 "serverUrl": "http://grafana:3000",
                 "orgId": "1",
-                "apiKey": "eyJrIjoiQ0c5NmZBOTR4NjY3VHJra3JEMzNneUlCSWJ4ejBGRTciLCJuIjoiUGVyZmFuYSIsImlkIjoxfQ==",
+                "apiKey": "apiKey",
```

Comment:
ja beter: geen default key in template

Reply:
The key is not a default. It will be replaced by a generated key anyway. It was commited by accident. I have changed it back to hte way it was It was to avoid complaints from GitGuard (which still complains by the way.